src/test/java/com/marklogic/client/ext/schemasloader/impl/ValidateTdeTemplatesTest.java

```diff
@@ -27,9 +27,20 @@ public void setup() {
 
 	@Test
 	public void badJsonFile() {
-		FailedRequestException ex = assertThrows(FailedRequestException.class, () ->
-			loader.loadSchemas(Paths.get("src", "test", "resources", "bad-schemas", "bad-json").toString()));
-		logger.info(ex.getMessage());
+		final String path = Paths.get("src", "test", "resources", "bad-schemas", "bad-json").toString();
```

Comment:
Need different assertions based on whether templateBatchInsert is supported or not.